frontend/src/metabase/dashboard/actions/utils.ts

```diff
@@ -57,3 +59,25 @@ export function haveDashboardCardsChanged(
     )
   );
 }
+
+export const getDashCardMoveToTabUndoMessage = (dashCard: StoreDashcard) => {
+  const virtualCardType =
+    dashCard.visualization_settings?.virtual_card?.display;
+
+  if (dashCard.card.name) {
+    return truncate(t`Card moved: ${dashCard.card.name}`, 75);
```

Comment:
I don't like using javascript to truncate text, because the information is lost. Usually in a CSS-truncated text, you could multiple-click the text to select and copy the whole text, but it won't work with this approach.

You should be able to achieve a similar effect using `max-width: 75ch`. Or just ballpark how long 75 characters would be and set it using `rem`

However if CSS approach is too hard to implement for any reason, I'd suggest you add `title` attribute to the text element as it's a good practice to have the full information in HTML, so screen readers can still read it. `title` also has a good default of showing the full untruncated text when hovering over the text.

Reply:
Design seems to be fine with truncating via css on all toast, solution has already been pushed to the PR in this commint: https://github.com/metabase/metabase/pull/35378/commits/004ba7b13943e40695242a663d1335ce158bf791